assets/base.css

```
@@ -248,7 +248,7 @@ html.no-js .no-js-hidden {
   padding: 0 1.5rem;
 }
 
-.page-width:not(.drawer-menu) {
+.page-width:not(:where(.drawer-menu)) {
```

I feel like it's even easier to read like this: 
```suggestion
.page-width:where(:not(.drawer-menu)) {
```

---

Good suggestion, Ludo. Thank you.